### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by kevinbett**
Adds templates for user sign up,  user login, business registration, business profile summary, business update, and listing catalog. 

Adds provisions for business reviews on the business profile template. 

Adds business delete option on the business update template. 

Adds provisions to allow filtered searches.

### CODE REVIEW

File: designs/UI/business_profile_update.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    </div>
+      <div class="button_holder" id="bt2">
+      <button type="submit" class="delbusiness">Delete Business</button>
+    </div>
+  </div>
+<!--</form>-->
+</div>
+
+</body>
+</html>
```
Comment (jmnyarega) [inline designs/UI/business_profile_update.html]: Add an extra line at the end of your file

File: designs/UI/business_registration.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Register Business | WeConnect</title>
+  <link rel="stylesheet" type="text/css" href="./css/businessreg.css"/>
+</head>
+<body>
+
+<div class="frame">
+<!--<form style="border:1px solid #ccc">-->
```
Comment (jmnyarega) [inline designs/UI/business_registration.html]: Avoid pushing commented code

File: designs/UI/user_registration.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Sign up | WeConnect</title>
+  <link rel="stylesheet" type="text/css" href="./css/userstyle.css"/>
+</head>
+<body>
+
+<div class="frame">
+<!--<form style="border:1px solid #ccc">-->
```
Comment (jmnyarega) [inline designs/UI/user_registration.html]: Avoid pushing commented code

File: designs/UI/business_profile_update.html (hunk 1 above)
Comment (kevinbett) [reply designs/UI/business_profile_update.html]: Thank you. Will make the amends.

File: designs/UI/business_registration.html (hunk 2 above)
Comment (kevinbett) [reply designs/UI/business_registration.html]: Will avoid such in future

File: designs/UI/user_registration.html (hunk 3 above)
Comment (kevinbett) [reply designs/UI/user_registration.html]: Thanks. This is noted.

File: designs/UI/business_registration.html (hunk 2 above)
Comment (kevinbett) [reply designs/UI/business_registration.html]: This has been rectified

**Comment by jmnyarega** [comment]
Still some of your commits are in past tense

**Pull request merged**